src/main/kotlin/com/jetbrains/snakecharm/inspections/quickfix/CreateMissedFile.kt

```diff
@@ -0,0 +1,142 @@
+package com.jetbrains.snakecharm.inspections.quickfix
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.application.ModalityState
+import com.intellij.openapi.command.undo.*
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.roots.ProjectRootManager
+import com.intellij.openapi.vfs.LocalFileSystem
+import com.intellij.openapi.vfs.VfsUtil
+import com.intellij.openapi.vfs.VirtualFileManager
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.jetbrains.snakecharm.SmkNotifier
+import com.jetbrains.snakecharm.SnakemakeBundle
+import com.jetbrains.snakecharm.lang.SnakemakeNames
+import org.apache.commons.io.FileUtils
+import java.io.IOException
+import java.nio.file.Files
+import java.nio.file.InvalidPathException
+import java.nio.file.Paths
+import kotlin.io.path.isDirectory
+import kotlin.io.path.name
+import kotlin.io.path.notExists
+
+class CreateMissedFile(
+    element: PsiElement,
+    private val fileName: String,
+    private val sectionName: String,
+    private val searchRelativelyToCurrentFolder: Boolean
+) : LocalQuickFixAndIntentionActionOnPsiElement(element) {
+    companion object {
+        private val condaDefaultContext = """
+        channels:
+        dependencies:
+    """.trimIndent()
+
+        // Update it if wee need default text
+        val supportedSections = mapOf(
+            SnakemakeNames.SECTION_CONDA to condaDefaultContext,
+            SnakemakeNames.SECTION_NOTEBOOK to null,
+            SnakemakeNames.SECTION_SCRIPT to null,
+            SnakemakeNames.MODULE_SNAKEFILE_KEYWORD to null,
+
+            SnakemakeNames.WORKFLOW_CONFIGFILE_KEYWORD to null,
+            SnakemakeNames.WORKFLOW_PEPFILE_KEYWORD to null,
+            SnakemakeNames.WORKFLOW_PEPSCHEMA_KEYWORD to null
+        )
+    }
+
+    override fun getFamilyName() = SnakemakeBundle.message("INSP.NAME.conda.env.missing.fix", fileName)
+
+    override fun getText() = familyName
+
+    override fun invoke(
+        project: Project,
+        file: PsiFile,
+        editor: Editor?,
+        startElement: PsiElement,
+        endElement: PsiElement
+    ) {
+        val relativeDirectory =
```

Comment:
Fix doesn't work ok if absolute path is used, e.g.:


```
rule rule277_s:
    conda: "/ednvs/ddd/ddd/dddaaaa.yaml"
    script: "env/ddddd.py"
```

on runtime expects absolute path:
```
$ snakemake --snakefile rule277.smk --use-conda -c1
Building DAG of jobs...
WorkflowError:
Failed to open source file /ednvs/ddd/ddd/dddaaaa.yaml
FileNotFoundError: [Errno 2] No such file or directory: '/ednvs/ddd/ddd/dddaaaa.yaml'
```

but quick fixes tries to create a file in project working directory. I assume that it valid for all file references for conda, script, etc sections. E.g. path could be absolute or relative. Is better to check.